### PR TITLE
Add deploy github action

### PR DESCRIPTION
**Pull request by MartinHjelmare**

### CODE REVIEW

**Comment by MartinHjelmare** [comment]
@balloob  from your comment in #38 I'm guessing there's some flaw here.

**Comment by balloob** [comment]
The action is perfect :) However we need an aws credential set up. We need to create one with just enough privileges to deploy this project, nothing more. I don't know how 🙈 Hoping @pvizeli can help here.

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - run: npm install
```
Comment (ludeeus) [inline .github/workflows/deploy.yml:17]: You should add a test step here so it does not deploy if tests fail.

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (MartinHjelmare) [reply .github/workflows/deploy.yml:17]: Yes! I'll wait with updating here until we have a solution for the credentials.

**Comment by MartinHjelmare** [comment]
I'll close here until we have solved the cloud requirements.

**Pull request closed without merging**